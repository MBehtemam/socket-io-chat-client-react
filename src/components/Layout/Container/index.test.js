import React from "react";
import styled from "styled-components";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Container from "./index";

describe("Test suits for Container", () => {
  test("it should match to snapshot", () => {
    const tree = renderer.create(<Container />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test("it should has correct styles", () => {
    const tree = renderer.create(<Container />).toJSON();
    expect(tree).toHaveStyleRule("display", "flex");
    expect(tree).toHaveStyleRule("flex-direction", "column");
  });
});
