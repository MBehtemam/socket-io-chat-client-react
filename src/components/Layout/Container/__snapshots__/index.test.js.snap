// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Container it should match to snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
}

<div
  className="c0"
/>
`;
